agent/utils/retry/exponential_backoff.go

```diff
@@ -0,0 +1,48 @@
+package retry
```

Comment:
i think you need to add copy right header for the new files in this package?

Reply:
and update the header of the modified files to xxx-2019 i guess

Reply:
Good catch! updated